src/main/scala/com/microsoft/ml/spark/lightgbm/LightGBMClassifier.scala

```diff
@@ -100,6 +102,7 @@ class LightGBMClassificationModel(override val uid: String)
     * @return transformed dataset
     */
   override def transform(dataset: Dataset[_]): DataFrame = {
+    logInfo("Calling function transform --- telemetry record")
```

Comment:
do you need "--- telemetry record"?  Also can you include the class name, like "Calling LightGBMClassificationModel transform"?

Reply:
Actually logInfo automatically fetches the classname, that's why I don't add it in the message.